opflexagent/vpplib/VPPApi.py

```diff
@@ -49,8 +49,7 @@ def __init__(self, client_name, LOG):
         self.connected = False
 
     def __enter__(self):
-        self.vppp = VppPapiProvider(self.client_name, self.LOG,
-            None, self.read_timeout)
+        self.vppp = VppPapiProvider(self.client_name, None, self.read_timeout)
```

Comment:
Wondering how this error can get away with it before.....Don't we have a UT in place to exercise this code path then?

Reply:
I have used mock APIs at this level in the UT intentionally. We were not sure about whether we would release this library or the VPP team. I had all these changes locally and missed committing  the last time around, which was why I retested this path.